src/components/ConnectionTimer.tsx

```diff
@@ -24,8 +30,12 @@ export const ConnectionTimer: React.FunctionComponent<IConnectionTimerProps> = (
       console.time("disconnected");
       setConnected(false);
     });
-    props.container.on("connected", () => {
+    props.container.on("connected", async () => {
       console.log("-- connected to document --");
+      const signaler = props.container.initialObjects.signaler as Signaler;
+      // Add the submitSignal use here to future reference.
+      signaler.submitSignal("connectRequest");
+      await delay(100);
```

Comment:
What is the purpose of this delay?

Reply:
Thanks for asking. Actually our throttler is not very sensitive, I add the delay function, so that we can add more submitOps after the throttler make the count to trigger the throttler.